source/custom/nidcpower_service.custom.cpp

```diff
@@ -1,5 +1,56 @@
 #include <nidcpower/nidcpower_service.h>
 
+#include <stdexcept>
+
 namespace nidcpower_grpc {
 
+struct DriverErrorException : std::runtime_error {
+  DriverErrorException(int status) : std::runtime_error("") { status_ = status; }
+  int status_ = 0;
+};
+
+void CheckStatus(int status)
+{
+  if (status != 0) {
+    throw DriverErrorException(status);
+  }
+}
+
+::grpc::Status NiDCPowerService::MeasureMultiple(::grpc::ServerContext* context, const MeasureMultipleRequest* request, MeasureMultipleResponse* response)
```

Comment:
The scope custom cpp file seems to add below two lines at the top of each API. They help visually identify where each of the function starts easily. Our generated cpp files have them too. Add them here as well for consistency.


//---------------------------------------------------------------------
//---------------------------------------------------------------------

Reply:
Made the changes. I missed this before.